crates/rolldown/tests/fixtures/tree_shaking/export-star/artifacts.snap.new

```diff
@@ -0,0 +1,21 @@
+---
```

Comment:
I don't see any difference compared to branch `fix/export-star-external`

Reply:
The `export-star` is not external..